frontend/src/metabase/public/lib/code-templates.ts

```diff
@@ -1,29 +1,48 @@
+import { match } from "ts-pattern";
+
 import { optionsToHashParams } from "./embed";
 import type {
   CodeSampleParameters,
   EmbeddingDisplayOptions,
+  EmbeddingHashOptions,
   EmbeddingParametersValues,
 } from "./types";
 
 export function getIframeQueryWithoutDefaults(
   displayOptions: EmbeddingDisplayOptions,
 ) {
+  const hashOptions = transformEmbeddingDisplayToHashOptions(displayOptions);
+
   return optionsToHashParams(
-    removeDefaultValueParameters(displayOptions, {
+    removeDefaultValueParameters(hashOptions, {
       theme: "light",
       hide_download_button: false,
       background: true,
     }),
   );
 }
+
+function transformEmbeddingDisplayToHashOptions(
+  displayOptions: EmbeddingDisplayOptions,
+): EmbeddingHashOptions {
+  const downloads = match(displayOptions.downloads)
+    .with(null, () => null) // do not include the "downloads" parameter at all, used for OSS.
+    .with({ pdf: true, dashcard: false }, () => "pdf")
```

Comment:
as @npretto has pointed out. I think this should be something else. We could continue to use `results` so it stays the same as our hash name.

Reply:
Agreed, fixed in https://github.com/metabase/metabase/pull/56211/commits/f8d9b329d3c79abe8682b39fe599031fb697b63b 🙏🏻